### ISSUE TITLE
Tribal Nations index vs. screenreader 

### ISSUE THREAD

**Issue by lfrancis** [usnationalarchives/nara-bia-photos#142]
https://nara-bia-photos.herokuapp.com/tribal-nations#m

When a VoiceOVer user activates a link in the index, while the page scrolls to that point, focus does not change to the activated anchor link, so the index is not accessible for screenreader users.

I would expect that if the screenreader elected M then focus would change to the M anchor and the M would be announced along with the number of tribes, and reading could continue at that point.

![image](https://user-images.githubusercontent.com/6942589/96001454-0b78f080-0e06-11eb-9900-5ac38d8c1108.png)
